File: Leetcode/ARRAY/JAVA/Easy Problems/P_1929.java



public class P_1929 {
   
    public static int[] concatination(int[] nums) {
        int[] ans = new int[2 * nums.length];
        for (int i = 0; i < nums.length; i++) {
            ans[i] =  ans[i+nums.length] = nums[i];
        }
        return ans;
    }

    // Main method
    public static void main(String[] args) {
        int[] nums = {1, 2, 3};
        int[] result = concatination(nums); 
       
        for (int i = 0; i < result.length; i++) {
            System.out.print(result[i] + " ");
        }
    }
}

File: Leetcode (DSA)/ARRAY/JavaScript/Easy Problems/P_1929.js



var getConcatenation = function (nums) {
    
    const n = nums.length;

    for (let i = 0; i < n; i++) {
        nums.push(nums[i]);
    }

    return nums;
};

let nums = [1, 2, 1];
console.log(getConcatenation(nums))
